### PR TITLE
Change poverty status threshold

### PR DESCRIPTION
**Pull request by tonyfujs**
* Switch < to <=
* remove unused line of code

### CODE REVIEW

**Comment by tonyfujs** [comment]
Fixes issue #158

**Comment by tonyfujs** [comment]
Hey @randrescastaneda 
Can you take a look at what I did here when you get a chance? This is a very small code change, but I want to make sure it makes sense from a methodology standpoint:

- It fixes the problem identified in issue #158 
- It did not break any of our existing unit tests
Thanks!

File: R/md_compute_poverty_stats.R [hunk 1]
```diff
@@ -24,8 +24,7 @@ md_compute_poverty_stats <- function(welfare, weight, povline_lcu) {
   welfare <- welfare[o]
   weight  <- weight[o]
 
-  alpha             <- c(0, 1, 2)
```
Comment (randrescastaneda) [inline R/md_compute_poverty_stats.R:27]: This is OK

File: R/md_compute_poverty_stats.R [hunk 2]
```diff
@@ -24,8 +24,7 @@ md_compute_poverty_stats <- function(welfare, weight, povline_lcu) {
...
-  pov_status        <- (welfare < povline_lcu)
+  pov_status        <- (welfare <= povline_lcu)
```
Comment (randrescastaneda) [inline R/md_compute_poverty_stats.R:27]: Poverty status should be one if welfare is less than the poverty lines, not less or equal to... 
image

**Review by randrescastaneda — CHANGES_REQUESTED** [review]
Hi @tonyfujs, 
Just a very small change needs to be made. Remove the = sign from the <= expression. 
Thanks. 
Best,

**Comment by randrescastaneda** [comment]
Hi @tonyfujs, 
I requested some changes but I did them myself, you don't need to do anything. I'll merge.

**Pull request merged**